acceptance/tests/catalog/replace_catalog_debugging.rb

```diff
@@ -92,5 +92,33 @@
 
   end
 
+  step "relationship on resource alias" do
+
+    manifest = <<MANIFEST
+
+    package { foo:
+      ensure => installed,
+      alias=>bar,
+    }
+    service{ baz:
+      ensure => running,
+      require => Package[bar]
+    }
+
+MANIFEST
+
+    run_agents_with_new_site_pp(master, manifest, {}, [0,2,4])
```

Comment:
@kbarber it seems like the el5 hosts are throwing a 6 here -- apparently signifies changes and failures. Any idea why there'd be a difference?